cmd/vcfFilter/vcfFilter.go

```diff
@@ -252,11 +256,23 @@ func getTests(c criteria, header vcf.Header) testingFuncs {
 				return true
 			})
 	}
+	if c.subSet < 1 {
+		answer = append(answer,
+			func(v vcf.Vcf) bool {
+				r := rand.Float64()
+				if r > c.subSet {
+					return false
+				}
+				return true
+			})
+	}
 	return answer
 }
 
 func main() {
 	var expectedNumArgs int = 2
+	var randSeed *bool = flag.Bool("randSeed", false, "Uses a random seed for the RNG.")
+	var setSeed *int64 = flag.Int64("setSeed", -1, "Use a specific seed for the RNG.")
```

Comment:
2 flags for seeds seems like overkill. How about if setSeed is -1 then it uses a random seed, else it uses setSeed.

Reply:
randSeed by default is pretty dangerous. If we want to move to one flag, I think we should abandon the randSeed all together, as it produces data that can never be replicated. I never use it and would suggest people don't

Reply:
You can default setSeed to 19 (or some other number) so that it always runs with a constant seed by default and is repeatable.  Then someone can use the setSeed option to run with another seed (other than 19) if they would like to get a different sample.  I think the question here is: do we make -1 a special value of set seed that uses a random seed, or do we have a separate option to do a random seed (or do we not allow random seeds).  I am not super opinionated on this.  I think Riley makes a good point that it is probably better programming/science practice to generate random numbers, and then use these random seeds/numbers as the starting point for the analysis with setSeed, but this may be beyond some users and "randSeed" would be a quick option for them when generating their files.

Reply:
Ah, yeah I can see how that would cause problems. Is there much value in providing a random seed option? If not, I would agree with Riley and just remove it. I don't see much reason to enable people to use a bad practice. If they really want a random seed in each run then they can wrap it in a bash script.